plugins/logitech-hidpp/fu-logitech-hidpp-peripheral.c

```diff
@@ -688,6 +689,7 @@ fu_logitech_hidpp_peripheral_detach (FuDevice *device, GError **error)
 			g_prefix_error (error, "failed to put device into DFU mode: ");
 			return FALSE;
 		}
+		g_usleep (200 * 1000);
```

Comment:
What is the context of this sleep?
It seems it is when putting the device in DFU, right? If so, seems reasonable.

Please note that there are 2 different hardware variants and one of them takes significantly longer to validate cryptographic signatures, so if we are waiting for the device to wake up after flashing, that needs to be taken into account. But that should already be implemented and I think this code is about putting it in DFU, so not relevant, just thought I would note this in case.